lib/screens/profile_screen.dart

```diff
@@ -197,23 +198,37 @@ class ProfileScreen extends StatelessWidget {
   }
 
   Future<void> signInWithApple() async {
-    final rawNonce = generateNonce();
-    final nonce = sha256ofString(rawNonce);
-
-    final appleCredential = await SignInWithApple.getAppleIDCredential(
-      scopes: [
-        AppleIDAuthorizationScopes.email,
-        AppleIDAuthorizationScopes.fullName,
-      ],
-      nonce: nonce,
-    );
-    // Create an `OAuthCredential` from the credential returned by Apple.
-    final oauthCredential = OAuthProvider('apple.com').credential(
-      idToken: appleCredential.identityToken,
-      rawNonce: rawNonce,
-    );
+    // Sign in the user with Firebase.
+    if (kIsWeb) {
+      final provider = OAuthProvider("apple.com")
+        ..addScope('email')
+        ..addScope('name');
+
+      await FirebaseAuth.instance.signInWithPopup(provider);
+    } else {
+      final rawNonce = generateNonce();
+      final nonce = sha256ofString(rawNonce);
+      var redirectURL = "https://andaxapp.firebaseapp.com/__/auth/handler";
+      var clientID = "andaxapp";
```

Comment:
Single quotes here as well. And also try to use `final` (or maybe even `const` in this case) instead of `var` whenever we do not need to reassign to the variable